modell/src/main/kotlin/no/nav/tilbakekreving/behandling/saksbehandling/Saksbehandlingsteg.kt

```diff
@@ -0,0 +1,23 @@
+package no.nav.tilbakekreving.behandling.saksbehandling
+
+import no.nav.tilbakekreving.FrontendDto
+import no.nav.tilbakekreving.kontrakter.behandling.Behandlingsstatus
+import no.nav.tilbakekreving.kontrakter.behandlingskontroll.Behandlingssteg
+import no.nav.tilbakekreving.kontrakter.behandlingskontroll.Behandlingsstegstatus
+
+internal interface Saksbehandlingsteg<FrontendDTO> : FrontendDto<FrontendDTO> {
```

Comment:
Ref: https://kotlinlang.org/docs/coding-conventions.html#choose-good-names
```suggestion
internal interface Saksbehandlingsteg<FrontendDto> : FrontendDto<FrontendDto> {
```